oslo_cache/backends/redis.py

```diff
@@ -0,0 +1,45 @@
+from dogpile.cache.backends import redis as redis_backend
+from oslo_cache import core
+__all__ = [
```

Comment:
```suggestion

__all__ = [
```
https://peps.python.org/pep-0008/#blank-lines